src/monexamine.cpp

```diff
@@ -263,6 +266,17 @@ int monexamine::pet_armor_pos( monster &z )
     return g->u.get_item_position( loc.get_item() );
 }
 
+int monexamine::tack_pos()
```

Comment:
This function could be static and not exposed in the header. It is not used anywhere else but in this cpp file.